### PR TITLE
Adding blogpost 10 - Xfce is moving from IRC to Matrix

### PR DESCRIPTION
**Pull request by alexxcons**

### CODE REVIEW

File: blogpost_10.html [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+            
+            <p>In the past, it was possible to connect and idle in libera IRC channels via <a href="https://matrix.org/ecosystem/bridges/irc/">Matrix.org bridge</a>.
+            Using Matrix to connect is very convenient for several reasons:</p>
+
+            <ul>
+                <li>When accessing IRC channels via Matrix bridge, it's not necessary to manage an <a href="https://en.wikipedia.org/wiki/BNC_(software)">IRC bouncer</a>.
+                    Bridged Matrix channels retain messages that arrive while you are offline.</li>
+                <li>Chat history is synchronised across multiple devices by default. No special BNC setup is required.</li>
+                <li>The entry barrier for Matrix is lower than for IRC.</li>
+                <li>On IRC, after answering a support question, you might realize that the person asking it went offline hours ago (no BNC used) and as such you wasted your time. Wont happen if the question was raised by a Matrix user.</li>
```
Comment (pwbriggs) [inline blogpost_10.html]: Apologies for nitpicking, 😄 but I think you're missing an `'` in `Wont` (here and in both the feed files).

**Comment by andreldm** [comment]
Looks good, thanks for writing this down, it's important to announce such things. Just two remarks:

1. "on IRC" vs "in IRC", I'm no native and not really good at grammar, but I suspect "on IRC" is the right one.
2. "The entry barrier for Matrix is lower than for IRC." I agree, but this is controversial/opinion-based affirmation. You can say this is our opinion or based on our own experience, it's written as a generally accepted fact.

File: blogpost_10.html (hunk 1 above)
Comment (alexxcons) [reply blogpost_10.html]: Thank you for the nitpick, it's very welcome! Fixed now.

**Comment by alexxcons** [comment]
> Looks good, thanks for writing this down, it's important to announce such things. Just two remarks:
> 
>     1. "on IRC" vs "in IRC", I'm no native and not really good at grammar, but I suspect "on IRC" is the right one.
> 
>     2. "The entry barrier for Matrix is lower than for IRC." I agree, but this is controversial/opinion-based affirmation. You can say this is our opinion or based on our own experience, it's written as a generally accepted fact.

Good, hint, thank you! Just changed it accordingly. Think I will wait another day for possible further change requests, and then publish it.

**Pull request merged**